app/ide-desktop/lib/dashboard/src/tailwind.css

```diff
@@ -237,6 +237,8 @@
 
   --menu-entry-padding: 0.25rem;
 
+  --delete-color: rgba(243 24 10 / 87%);
```

Comment:
`rgba` -> `rgb`

Reply:
Not sure we can easily replace this color with RGB only alternative, since it heavily depends on a background. Actually, I copied this color(and all below) from tailwind.config.js so I guess we can leave it as is?

Reply:
`rgba` is an obsolete syntax,  `rgb(243 24 10 / 87%);` is completely valid